src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -170,7 +170,8 @@ class EmojiPickerMenu extends Component {
 
             // Return if the key is related to any tab cycling event so that the default logic
             // can be executed.
-            if (keyBoardEvent.key === 'Tab' || keyBoardEvent.key === 'Shift' || keyBoardEvent.key === 'Enter') {
+            if (keyBoardEvent.key === 'Tab' || keyBoardEvent.key === 'Enter'
+                || (keyBoardEvent.key === 'Shift' && this.searchInput && !this.searchInput.isFocused())) {
```

Comment:
I feel like this condition is quite complex that it would be handy to improve the comment a bit to explain why its like this